coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalHierarchyServiceImpl.java

```diff
@@ -349,6 +349,18 @@ public void synchronizeAllChildren(DevelopmentProposal hierarchyProposal) throws
         prepareHierarchySync(hierarchyProposal);
         synchronizeAll(hierarchyProposal);
     }
+    
+    public void synchronizeAllChildrenBudgets(DevelopmentProposal hierarchyProposal) throws ProposalHierarchyException {
```

Comment:
You mean create a new service in that package? We haven't ever done that but could I guess. There are some related functionality we'd have to copy though. Like the deepCopy method. And either way the propdev version will still deal with some minor budget interactions.